spec/factories/courses.rb

```diff
@@ -1,4 +1,12 @@
 FactoryGirl.define do
+  factory :user do 
+    first_name "John"
+     last_name "Smith"
+     email "js@email.com"
+     # course_id course.id
```

Comment:
<img width="611" alt="screen shot 2017-06-22 at 7 49 14 pm" src="https://user-images.githubusercontent.com/24990450/27460749-f3219bfe-5783-11e7-84ba-48ae27a7817d.png">

Using it in course_spec.rb